agent/agent-profiler/agent-alerting-api/build.gradle.kts

```diff
@@ -1,3 +1,7 @@
 plugins {
   id("ai.java-conventions")
 }
+
+dependencies {
+  implementation("com.fasterxml.jackson.core:jackson-annotations:2.13.3")
+}
```

Comment:
is this still needed?

Reply:
unfortunately yes, in order to serialize the enums correctly we need access to the JsonProperty annotation

Reply:
sorry, I think I'm missing something, where does the serialization of AlertingConfig happen?

Reply:
That serialization happens in the diagnostics so is not in this repository. However you have just reminded me I can use Jacksons mixins to configure the serialization without modifying this class, will remove it from here and use mixins to get the serialization right

Reply:
> will remove it from here and use mixins

that sounds nice, but if it becomes any trouble, it's ok to keep the annotations here, just add a comment to explain their usage so we don't go removing them thinking they're unused 😅